taca/analysis/analysis.py

```diff
@@ -40,11 +40,11 @@ def _run_type(run):
         #this information about the run type (with HiSeq2.5 applicationaName does not work anymore, but as for a long time we will have instruments not updated I need to find out somehting that works
         try:
             #Works for recent control software
-            runtype=rp.data['RunParameters']["Setup"].get("Flowcell")
+            runtype=rp.data['RunParameters']["Setup"]["Flowcell"]
         except KeyError:
             #use this as second resource but print a warning in the logs
             logger.warn("Parsing runParameters to fecth instrument type, not found Flowcell information in it. Using ApplicaiotnName")
-            runtype=rp.data.get("ApplicationName")
```

Comment:
@Galithil I checked this, should work with existing runParameters.xml on HiSeq, MiSeq, and HiSeqX all of them have RunParameters -> Setup -> ApplicationName
However, in the case of HiSeqX v2.5 Flowcell needs to be used to understand we are running an X flowcell. 

About the empty string... there is no big difference, the if clause at the end of the function will return None in case runtype is empty string.